src/components/Invites/InvitesPageLayout.tsx

```
@@ -0,0 +1,61 @@
+                    )}
+                >
+                    {/* render animated star decorations */}
+                    {STAR_POSITIONS.map((positions, index) => (
+                        <Image
+                            src={starImage.src}
+                            alt="star"
+                            width={56}
+                            height={56}
+                            className={twMerge(positions, 'absolute z-10')}
+                            priority={index === 0}
+                        />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)